src/it/resources/com/google/checkstyle/test/chapter2filebasic/rule231filetab/InputFormattedWhitespaceCharacters.java

```diff
@@ -0,0 +1,39 @@
+package com.google.checkstyle.test.chapter2filebasic.rule231filetab;
+
+final class InputFormattedWhitespaceCharacters {
+  // Long line ----------------------------------------------------------------
+  // Contains a tab ->	<- // violation 'Line contains a tab character.'
+  // Contains trailing whitespace ->
```

Comment:
and here too

Reply:
issue is created https://github.com/google/google-java-format/issues/1149